quick-start/src/main/java/com/marklogic/quickstart/service/MappingManagerService.java

```diff
@@ -141,14 +90,13 @@ public void deleteMapping(String mapping) throws IOException {
     }
 
     public MappingModel getMapping(String mappingName) throws IOException {
-        List<MappingModel> mappings = getMappings();
-
-        for (MappingModel mapping : mappings) {
-            if (mapping.getName().equals(mappingName)) {
-                return mapping;
-            }
+        mappingManager = MappingManager.getMappingManager(envConfig().getMlSettings());
+        try{
+           ObjectMapper objectMapper = new ObjectMapper();
+            return MappingModel.fromJson(objectMapper.readTree(mappingManager.getMappingAsJSON(mappingName, -1)));
```

Comment:
heres the other side of my comment above -- mapper.convertValue( ..., MappingModel.class)

also ObjectMapper is best as a singleton object field.